lesson21/src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="ja" class="text-gray-500">
```

Comment:
Q
Why not also specify the font color in the body tag? In general, I felt that it should be specified in the body tag, not in the html tag.

Reply:
Thank you for your comment!
The TailwindCSS documentation recommends this method if you want to apply global styles to html or body elements.
It seems that you should add it to either html or body.

[Link](https://tailwindcss.jp/docs/adding-base-styles#html)

Reply:
That's right, I learned something!